README.md

```diff
@@ -15,29 +15,39 @@ Before running DICOMFuse, ensure that you've installed the following
 applications:
 
 *   Java SE Runtime Environment 8.
-*   [WinFps](https://github.com/billziss-gh/winfsp) for using FUSE in Windows.
+*   [WinFps](https://github.com/billziss-gh/winfsp) for using FUSE in Windows 
+    (Windows 7x64, Windows 10x64).
 *   [OSXFuse](https://osxfuse.github.io/) for using FUSE in macOS (macOS 10.14
     Mojave).
+*   [libfuse](https://github.com/libfuse/libfuse) for using FUSE in Linux, install 
+    libfuse - `sudo apt install libfuse2`
 
-> [libfuse](https://github.com/libfuse/libfuse) has been supported in Linux
-> kernel since 2.6.14 version.
+> FUSE has been supported in Linux kernel since 2.6.14 version.
 
 ### Credentials
 
 You can access the Cloud Healthcare API using
 [Google application default credentials](https://cloud.google.com/docs/authentication/production#howtheywork).
+Set Healthcare DICOM Editor role for 
+[service account](https://cloud.google.com/healthcare/docs/how-tos/authentication).
 You can set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, or you
 can use `--keyFile` option when you launch DICOMFuse.
 
+> More info about [roles](https://cloud.google.com/healthcare/docs/concepts/access-control#roles).
```

Comment:
Can remove this after applying my above suggestion.

Reply:
Done